.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,39 @@
+
+{
+	"name": "Codespaces for Thanos",
+
+	"context": "..",
+
+	"dockerFile": "",
```

Comment:
What does it mean if we don't specify dockerfile, what image will it use?

Reply:
https://github.com/thanos-io/thanos/pull/5570#discussion_r939927590

Reply:
If using the default image then let's delete this line to avoid confusion

Reply:
@yeya24, is it possible to sign DCO directly from github 🤔